test/SpecHelper.hs

```diff
@@ -61,3 +63,21 @@ getEllipsedParamNameR total per elements pageParamName = do
 
 withApp :: SpecWith (TestApp App) -> Spec
 withApp = before $ pure (App, id)
+
+shouldIncludeAll
```

Comment:
This was needed for the props "doesn't remove not-ours elements" and "doesn't add not-ours elements". It was suggested as an addition to `hspec-expectations` in ["this PR"](https://github.com/hspec/hspec-expectations/pull/43) but hasn't been merged.